src/css/components/header.scss

```diff
@@ -16,6 +16,15 @@
   padding-bottom: $grid-3;
 }
 
+// kludge for targeting dashboard
+.js-app {
```

Comment:
Rather then add more kludge, why don't we add another header class, like `.header-lg` or `header-full`.